README.md

```diff
@@ -137,7 +136,7 @@ art.users.unlock("test_user")
 
 Get the list of groups:
 ```python
-groups = art.groups.list()
+groups = art.groups.list_all()
```

Comment:
I personally prefer `list()` but I'd be happy to hear your motivations for this change

Reply:
reverted